src/index.ts

```diff
@@ -1,7 +1,9 @@
 import Phaser from 'phaser';
 import configuration from './config';
+import DemoScene from './scenes/DemoScene';
 import GameScene from './scenes/GameScene';
+import WelcomeScene from './scenes/WelcomeScene';
 
 const game = new Phaser.Game(
-  Object.assign(configuration, { scene: [GameScene] })
+  Object.assign(configuration, { scene: [WelcomeScene, DemoScene, GameScene] })
```

Comment:
❓ Do we still need to load the base GameScene?

Reply:
Not sure lets try and remove  it
```suggestion
  Object.assign(configuration, { scene: [WelcomeScene, DemoScene] })
```